webhook/webhook.py

```diff
@@ -137,3 +142,68 @@ def handle_team_event(self, payload):
             logging.error("invalid payload received: {}".
                           format(str(payload)))
             return "invalid payload", 405
+
+    def handle_membership_event(self, payload):
+        """
+        Handle when a user is added, removed, or invited to an organization.
+
+        If the member is removed, they are removed as a user from rocket's db
+        if they have not been removed already.
+
+        If the member is added or invited, do nothing.
+        """
+        action = payload["action"]
+        github_user = payload["member"]
+        github_username = github_user["login"]
+        github_id = github_user["id"]
+        team = payload["team"]
+        team_id = team["id"]
+        team_name = team["name"]
+        selected_team = self.__facade.retrieve_team(team_id)

```

Comment:
I recently updated the dynamodb and facade classes to be less messy, and abstracted the class methods that had near identical functionality. Which means that the function `Facade.retrieve_team()` is no longer available. It is instead replaced by the more generic `Facade.retrieve()`.

So I would replace that line with:

```python
from models.team import Team
selected_team = self.__facade.retrieve(Team, team_id)
```

For more information, [check out the docs for this][docs-for-this].

Yeah. Basically, all the `_user` and `_team` and `_project` suffixes should be dropped in favour of the newer thing.

[docs-for-this]: https://rocket2.readthedocs.io/en/latest/docs/api/db.html

Reply:
Ok yeah I'll change those, they are from the latest version of master right?